modules/fscloud/variables.tf

```diff
@@ -65,3 +65,43 @@ variable "cbr_rules" {
   description = "(Optional, list) List of CBR rules to create."
   default     = []
 }
+
+########################################################################################################################
+# COS
+########################################################################################################################
+
+variable "destination_name" {
+  type        = string
+  description = "The Destintion name."
+  default     = null
+}
+
+variable "bucket_name" {
+  type        = string
+  description = "The bucket name in IBM cloud object storage instance."
+  default     = null
```

Comment:
update var description as per previous comment, and remove default value (making it a required variable)